src/main/java/fi/otavanopisto/kuntaapi/server/discover/EntityUpdater.java

```diff
@@ -0,0 +1,12 @@
+package fi.otavanopisto.kuntaapi.server.discover;
+
+@SuppressWarnings("squid:S1610 ")
+public abstract class EntityUpdater {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert the abstract class "EntityUpdater" into an interface. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1610)